docs/QSG/scratch.md

```diff
@@ -1,288 +1,297 @@
-Build from scratch
-==================
+# Build from scratch
```

Comment:
I personally don't think that it makes sense to document in the quickstart guide how to build from scratch. Anyway, i can move it to a separate section.